engine/survival_svm.py

```diff
@@ -247,6 +247,19 @@ def run(self):
 class PreprocessDataState(BlankState):
 
     def run(self):
+        self.update(message='Check survival times for zero and negative timepoints', progress=0.08)
+        split_manager = self.app.internal.get('split_manager')
+        for split in split_manager:
+            if not split.data.get('opt_out'):
+                model: LocalTraining = split.data.get('model')
+
+                dropped_rows = model.data.drop_negative_and_zero_timepoints()
+                if dropped_rows > 0:
+                    self.update(
+                        message=f'Dropped {dropped_rows} samples with zero or negative timepoints in {split.name}',
+                        state=STATE_ERROR
```

Comment:
I think we should use the "warning" state here. The error indicates that the app can not successfully finish. But here, we take care of the problems, which would make a warning more appropriate.

Reply:
Not sure how to set the warning state, as https://featurecloud.ai/assets/api/redoc-static.html#operation/status lists the status as an enum of the states `running`, `error`, `action_required`.
Is there an extra `warning` state or should I keep it in the `running` state than?

Reply:
I see. I suggest to keep it in running then.